frontend/src/components/RunInfo/RunInfoSection.js

```diff
@@ -0,0 +1,85 @@
+const RunInfoSection = (props) => {
+  let ago = (Date.now() - Date.parse(props.runInfo.started)) / 1000;
```

Comment:
Now that we have some package management I suspect you should be able to find some humanize JS library that does this better than my silly hand rolled one as well. I think GitHub has a library they published for doing this maybe?